packages/gasket-cli/src/commands/create.js

```diff
@@ -61,7 +62,17 @@ class CreateCommand extends Command {
         await loadPreset(context);
         cliVersion(context);
         applyPresetConfig(context);
-        await globalPrompts(context);
+        if (ciConfig) {
+          const parsedCiConfig = JSON.parse(ciConfig);
+          Object.assign(context, parsedCiConfig);
+          await readConfig(context);
+        } else if (ciConfigFilePath) {
+          const ciConfigFile = require(ciConfigFilePath);
+          Object.assign(context, ciConfigFile);
+          await readConfig(context);
```

Comment:
Tune up your readConfig to handle either the config json string or config file and run that action early before globalPrompts.